rust/operator-binary/src/spark_controller.rs

```diff
@@ -319,12 +321,9 @@ fn build_rolegroup_statefulset(
     rolegroup_ref: &RoleGroupRef<SparkCluster>,
     rolegroup_config: &HashMap<PropertyNameKind, BTreeMap<String, String>>,
 ) -> Result<StatefulSet, Error> {
-    let version = version(spark)?;
+    let version = image_version(spark)?;
```

Comment:
Naming it `image_version` or `spark_image_version` would be great

Reply:
done